app/src/components/keyCloakServiceClientMgr.js

```diff
@@ -151,20 +151,16 @@ class KeyCloakServiceClientManager {
     }
   }
 
-  async findUser(username) {
-    log.debug('KeyCloakServiceClientManager.findUser', username);
-    if (!username) {
-      log.error('KeyCloakServiceClientManager.findUser', 'No user provided.');
-      throw new Error('Cannot get a user in KeyCloak realm: username required.');
-    }
+  // Get all users based on search param, supply undefined as searchParams to get ALL users in realm
+  async findUsers(searchParams) {
```

Comment:
I like the use of parameters as this function could be handy for retrieving a bigger subset of users for example in an admin stats page

Reply:
Indeed. Find available query params here https://www.keycloak.org/docs-api/5.0/rest-api/index.html#_users_resource